src/main/java/eu/arrowhead/blacklist/BlacklistConstants.java

```diff
@@ -0,0 +1,73 @@
+package eu.arrowhead.blacklist;
+
+import java.util.List;
+
+import eu.arrowhead.common.jpa.ArrowheadEntity;
+
+public final class BlacklistConstants {
+
+	//=================================================================================================
+	// members
+
+	// System info related
+	public static final String SYSTEM_NAME = "blacklist";
+	public static final String VERSION_DISCOVERY = "1.0.0";
+	public static final String VERSION_MANAGEMENT = "1.0.0";
+	public static final String VERSION_GENERAL_MANAGEMENT = "1.0.0";
+	public static final String VERSION_MONITOR = "1.0.0";
+	public static final String METADATA_KEY_UNRESTRICTED_DISCOVERY = "unrestricted-discovery";
```

Comment:
This key should be moved to the common lib and that one should be used also in SR (and in the others too)